server/upackages.pas

```diff
@@ -179,14 +179,13 @@ procedure LoadPackageOrProject(const FileName: string);
     end;
   end;
 
-  procedure LoadPaths;
+  procedure LoadPaths(CompilerOptions: TDomNode);
   var
-    CompilerOptions, SearchPaths: TDomNode;
+    SearchPaths: TDomNode;
   begin
     Package.Paths.IncludePath := Package.Dir;
     Package.Paths.UnitPath    := Package.Dir;
 
-    CompilerOptions := Root.FindNode('CompilerOptions');
     if not Assigned(CompilerOptions) then
```

Comment:
Move the `nil` check to the parent function (see below).

Reply:
Done